tests/rules/select-style.test.ts

```diff
@@ -15,25 +15,25 @@ ruleTester().run(path.parse(__filename).name, rule, {
     import { Store } from '@ngrx/store'
```

Comment:
Something that came to my mind... since we have some rules relying on `Store` import, wouldn't it be a good idea to have a rule like `use-consistent-global-import` or something like this? Because if I do `import { Store as ngrxStore } from '@ngrx/store'` these rules won't work as expected...